drivers/sensor/aht20/aht20.c

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright (c) 2022 Rafael Lee <rafaellee.img@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT aosong_aht20
```

Comment:
Please stick to the product details asair, aht20 according to datasheet.

Reply:
I prefer to think that ASAIR is a brand name.
I'm sure that aosong is the company's name.

What's your suggesstion, use the company's name or the brand name?

AHT20 is listed here
[http://www.aosong.com/productslist-1.html](http://www.aosong.com/productslist-1.html)
[http://www.aosong.com/products-61.html](http://www.aosong.com/products-61.html)


DHT humidity and temperature sensor with which with driver in Zephyr.
[http://www.aosong.com/productslist.aspx?id=1&page=2](http://www.aosong.com/productslist.aspx?id=1&page=2)
[http://www.aosong.com/products-104.html](http://www.aosong.com/products-104.html)

zephyr/drivers/sensor/dht/dht.c
```C
#define DT_DRV_COMPAT aosong_dht
```

Reply:
It should be the company name. Is there a binding for this sensor in the Linux kernel?

Reply:
> It should be the company name. Is there a binding for this sensor in the Linux kernel?

Aosong is the company name.
I don't know whether those sensor drivers exist in Linux kernel. I mean there is a driver from the same company in the Zephyr repository already.
[https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/sensor/dht/dht.c](https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/sensor/dht/dht.c)